content/documentation/getting-started/index.adoc

```diff
@@ -63,51 +66,88 @@ If the Kiali version packaged with Istio or Maistra meets your needs then instal
 
-=== Install the Kiali Operator
 
-The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
+=== Install Kiali Operator
+
+If you did not install Kiali with Istio or Maistra, or want to replace the installed version of Kiali, it is done using the Kiali Operator. The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
+
+All of the installation options below:
 
-To install the Kiali Operator and, by default, the Kiali CR (custom Resource), run the install script via this command:
+* Ask to delete an existing Kiali Operator, or Kiali installation, when appropriate.
+* Prompt for the authentication strategy to use and/or additional information needed for login.
+** See link:#_login_options[Login Options] for more on the `login`, `anonymous` and `openshift` authentication strategies.
+
+
+==== Default Installation
+
+This option installs the Kiali Operator and the Kiali CR (Custom Resource), with default options.  Kiali will be able to access existing namespaces, but will require a CR update to access future namespaces (for more, see link:#_namespace_management[Namespace Management]. Run the install script via this command:
 
 [source,bash]
 ----
 bash <(curl -L https://git.io/getLatestKialiOperator)
 ----
 
-This will pull the latest version of the Kiali Operator and Kiali itself, and install them using the default options. If Kiali was already installed, you will be prompted to remove the older version.
 
-icon:lightbulb[size=1x]{nbsp} Environment variables or command-line options affect the Kiali Operator install script.  For example, if you do not want the script to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script.  Or run the script with "-oik=false" Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available.  Alternatively, use the script's "--help" command line option for help documentation.
+==== Demo/Dev Installation
 
-icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+Like the Default Installation but Kiali will be able to access existing namespaces and future namespaces. Because this grants cluster roles it is not recommended for production. Run the install script via this command:
```

Comment:
```suggestion
Like the Default Installation but Kiali will be able to access existing namespaces and future namespaces. Because this grants special cluster role permissions, it is not recommended for production. Run the install script via this command:
```

Reply:
+1